### PR TITLE
Sb 3732 - External url preview integration

### PR DESCRIPTION
**Pull request by revathitarento**

### CODE REVIEW

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts [hunk 1]
```diff
@@ -178,7 +180,7 @@ describe('CoursePlayerComponent', () => {
     spyOn(courseConsumptionService, 'getCourseHierarchy').and.returnValue(Observable.of(CourseHierarchyGetMockResponse.result.content));
     spyOn(courseConsumptionService, 'getContentStatus').and.returnValue(Observable.of(CourseHierarchyGetMockResponse.result));
     spyOn(courseConsumptionService, 'getConfigByContent').and.returnValue(Observable.of(CourseHierarchyGetMockResponse.result));
-      component.ngOnInit();
+    component.ngOnInit();
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts]: Format

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts [hunk 2]
```diff
@@ -196,7 +198,7 @@ describe('CoursePlayerComponent', () => {
...
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts]: Format

File: src/app/client/src/app/modules/resource/components/content-player/content-player.component.ts [hunk 3]
```diff
@@ -73,6 +73,10 @@ export class ContentPlayerComponent implements OnInit {
    */
   createNoteData: INoteData;
   closeUrl: any;
+  /**
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/resource/components/content-player/content-player.component.ts]: Format

File: src/app/client/src/app/modules/shared/components/player/player.component.ts [hunk 4]
```diff
@@ -44,6 +47,13 @@ export class PlayerComponent implements OnInit, OnChanges {
       }
     });
   }
+  /**
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/shared/components/player/player.component.ts]: Format

File: src/app/client/src/app/modules/shared/components/redirect/redirect.component.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { ResourceService } from '../../services';
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-redirect',
+  templateUrl: './redirect.component.html',
+  styleUrls: ['./redirect.component.css']
+})
+/**
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/shared/components/redirect/redirect.component.ts]: Typos in this comment.

'Redirectcomponent is called ...' change to 'Redirect component is called ...'
'lean/redict' change to 'learn/redirect'

File: src/app/client/src/app/modules/shared/services/content-utils/content-utils.service.ts [hunk 6]
```diff
@@ -56,4 +59,20 @@ export class ContentUtilsServiceService {
     }
     return this.baseUrl + 'play' + '/' + playertype + '/' + identifier;
   }
+  /**
+   * getRedirectUrl
```
Comment (bvinayakumar) [inline src/app/client/src/app/modules/shared/services/content-utils/content-utils.service.ts]: Comment is not proper.

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts (hunk 1 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts]: done

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts (hunk 2 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts]: done

File: src/app/client/src/app/modules/resource/components/content-player/content-player.component.ts (hunk 3 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/resource/components/content-player/content-player.component.ts]: done

File: src/app/client/src/app/modules/shared/components/player/player.component.ts (hunk 4 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/shared/components/player/player.component.ts]: done

File: src/app/client/src/app/modules/shared/components/redirect/redirect.component.ts (hunk 5 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/shared/components/redirect/redirect.component.ts]: done

File: src/app/client/src/app/modules/shared/services/content-utils/content-utils.service.ts (hunk 6 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/shared/services/content-utils/content-utils.service.ts]: Done

**Pull request merged**